### PR TITLE
My-Portfolio

### PR DESCRIPTION
**Pull request by zakiyousufi**
After the accepting first review >>

- I created the whole portfolio project  mobile version
- My project is responsive (Accepting any width size of the screen without scrolling)
- I followed the project requirements 
- there is no linters error in my project 
Thank you

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -117,3 +117,284 @@ nav {
...
+}
+
+.language-list-content {
+  margin: 30px 30px 0  0;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+}
```
Comment (vic778) [inline style.css]: - [ ] Kindly the `li` are not aligned center as you can see at the screen bellow, you can `flex-direction: column` and `gap: 32px;` the parent then you `centered` the children.  Fix that and let's see your project approved :handshake: 

Your design
Screenshot from 2022-05-28 19-59-14

Figma design
Screenshot from 2022-05-28 20-01-05

File: index.html [hunk 2]
```diff
@@ -65,6 +68,266 @@ <h2 class="my-position">I’am a Software Developer</h2>
...
+                                <img class="tweeter icons" alt="Tweeter Icon" src="css/Icons/social_media/tweeter.png">
+                            </a>
+                        </li>
+                        <li class="img-list">
+                            <a href="#" target="_blank" rel="noopener">
+                                <img class="m-icon icons" alt="Social Media Icon" src="css/Icons/social_media/M.png">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
```
Comment (vic778) [inline index.html]: - [ ] Kindly align center the icons. You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Design
Screenshot from 2022-05-28 20-06-37

Figma Design

Screenshot from 2022-05-28 20-07-07

**Review by vic778 — CHANGES_REQUESTED** [review]
Hello @zakiyousufi 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jaflih — APPROVED** [review]
Hi @zakiyousufi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
All linters are passing ✔️
Good commit history ✔️
Good PR description ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zakiyousufi** [comment]
@[jaflih](https://github.com/jaflih)
Thank you so much for your helpful feedbacks

**Pull request merged**